### PR TITLE
Blog App: Views

### PR DESCRIPTION
**Pull request by Mutalenic**
In this project, I have added the Views associated with the Models and Controllers I created for my Blog App.
## Built with
 - Ruby
 - Rubocop
 - Stylelint
 - RoR
## Made sure that
 - there are no linter errors.
- I used the correct Gitflow.
 - I documented my work in a professional way.
 - I used Ruby best practices.

### CODE REVIEW

File: app/views/posts/show.html.erb [hunk 1]
```diff
@@ -1,2 +1,10 @@
...
+<main>
+<section class="postscontainer">
+<div class="postcomment">
+  <h4><%= @post.title %></h4>
+  <p class="commentcounter">Comments: <%= @post.comments_counter || 0%>, Likes: <%= @post.likes_counter || 0%></p>
+  <p><%= @post.text %></p>
+  </div>
+  <%= render partial: 'comment', locals: {post: @post, comments: @post.comments}%>
+  </section>
+  </main>
```
Comment (thecodechaser) [inline app/views/posts/show.html.erb]: - [ ] On the post show page, Please display the author name and post number as available on the wireframe `post #1 by username`. Your view is missing that

### See the image for reference 

image

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1,2 +1,31 @@
...
+  <div class="posts">
+    <h4><%= post.title %></h4>
+    <p><%= post.text %> </p>
+    <span>
+      Comment: <%= post.comments_counter%>, Likes: <%= post.likes_counter || 0 %>
+    </span>
+  </div>
+  <% end %>
+  <% end %>
+</section>
```
Comment (thecodechaser) [inline app/views/users/show.html.erb]: - [ ] On the user show page, Please display the post number as it is available on the wireframe `post #1`. Your view is missing that.

### See the image for reference 

image

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -1,2 +1,15 @@
...
+    <p>Number of posts: <%= @user.posts_counter%> </p>
+  </div> 
+</section>
+
+<section class="usercontainer">
+  <% @user.posts.each do |post| %>
+    <%= render partial: 'post', locals: {post: post} %>
+  <% end %>
+</section>
+</main>
```
Comment (thecodechaser) [inline app/views/posts/index.html.erb:15]: - [ ] On the post index page, Please display the post number as it is available on the wireframe `post #1`. Your view is missing that.

### See the image for reference 

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Mutalenic👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/show.html.erb (hunk 1 above)
Comment (Mutalenic) [reply app/views/posts/show.html.erb]: @thecodechaser Thanks for the review, issue resolved.

File: app/views/users/show.html.erb (hunk 2 above)
Comment (Mutalenic) [reply app/views/users/show.html.erb]: @thecodechaser Thanks for the review, issue resolved.

File: app/views/posts/index.html.erb (hunk 3 above)
Comment (Mutalenic) [reply app/views/posts/index.html.erb:15]: @thecodechaser Thanks for the review, issue resolved.

File: app/views/posts/show.html.erb [hunk 4]
```diff
@@ -1,2 +1,11 @@
...
-<p>Show one post for a given user</p>
+<main>
+<section class="postscontainer">
+<div class="postcomment">
+<h4>Post<%= @post.id%><h4>
+  <h4><%= @post.title %></h4>
+  <p class="commentcounter">Comments: <%= @post.comments_counter || 0%>, Likes: <%= @post.likes_counter || 0%></p>
+  <p><%= @post.text %></p>
+  </div>
+  <%= render partial: 'comment', locals: {post: @post, comments: @post.comments}%>
```
Comment (codepantha) [inline app/views/posts/show.html.erb]: - [ ] According to the previous reviewer's suggestion, please include the username of the author in the post show page
### Requirement
image
### Your design
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUESTED CHANGES ♻️ 

## Hi @Mutalenic 👋, 

Well done implementing some of the required changes 👏, however some have been left unattended.
There are some issues that you still need to work on to go to the next project but you are almost there!

### Highlights 🎯 
✔️ Descriptive PR
✔️ Good commit history
✔️ App working as expected

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/show.html.erb (hunk 4 above)
Comment (Mutalenic) [reply app/views/posts/show.html.erb]: Thanks for the review @codepantha , issue has been resolved

**Review by Mre55 — APPROVED** [review]
**Hi @Mutalenic,**

# Status: APPROVED 🏆 🎉 🎊 

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 

Alt Text


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**